doc/modules/neural_networks_supervised.rst

```diff
@@ -0,0 +1,255 @@
+
+  * **Setting weight_scale**:  this controls regularization of the weights 
+    the same way as ``C`` does; but instead of regularizing the hidden-to-output 
+    layer weights, it regularizes the input-to-hidden layer weights.
+
+  * **Setting class_weight**: if the dataset contains an imbalanced ratio 
+    of class fequencies, it is desirable to set `class_weight='auto'`, as
+    it assigns more weight to the class having less number of samples.
+
+  * **Setting batch_size**: training on large datasets or having a large 
+    ``n_hidden`` value could cause issues for limited memory. In this case, it is 
+    recommended to set ``batch_size=1000`` in order to process the data in chunks. 
+    This will run the internal sequential learner which avoids computing the non-linear 
+    feature expansion on the full training set at once. Please note that the end 
+    results are the same regardless of the ``batch_size`` value but smaller values could 
+    increase training time.
+
+  * **Preprocessing**: Making sure that the input features vary on the same scale is 
+    important for having consistent prediction results. ``StandardScaler`` and ``**MinMaxScaler``
+    are two appropriate scalers for this task (see :ref:`link <preprocessing_scaler>`). 
+    Also, a good practice is to use ``Pipeline`` as demonstrated in 
+    :ref:`plot_elm_hyperparameters.py <example_neural_networks_plot_elm_hyperparameters.py>`. 
+
+Please see the example below to observe the impact of ``C`` and ``weight_scale`` 
+on testing score.
+
+.. figure:: ../auto_examples/neural_networks/images/plot_elm_hyperparameters_001.png
+   :target: ../auto_examples/neural_networks/plot_elm_hyperparameters.html
+   :align: center
+   :scale: 100%
+
+.. topic:: Examples:
+
+   * :ref:`example_neural_networks_plot_elm_hyperparameters.py`
+
+
+Mathematical formulation
+========================
+
+A standard ELM trains a single-hidden layer feedforward network using the 
+following function,
+
+:math:`y = \beta \cdot \sigma(W^T \cdot X + b)`
+
+where matrices ``X`` and ``y`` represent the input samples and target values, 
+respectively; matrices ``W`` and ``b`` are randomly generated based on a 
+uniform distribution; matrix :math:`\beta` contains unknown variables;
+and :math:`\sigma` is the non-linear, component-wise activation function.
+
+ELM solves for :math:`\beta` using the ridge regression implementation, given
+as,
+
+:math:`(H^T H + (1 / C) * I)^{-1} H^T y`
+
+where  :math:`H = \sigma(W^TX + b)`, ``C`` is a regularization term which controls the 
+linearity of the decision function, and ``I`` is the identity matrix.
+
+
+Related Models
+==============
+
+  * :ref:`Kernel approximation <kernel_approximation>` : this introduces other models
+    that map the input feature space to an intermediate non-linear feature space 
+    suitable for training linear models.
+
+.. topic:: References:
+
+ *  Huang, Guang-Bin, Qin-Yu Zhu, and Chee-Kheong Siew. 
```

Comment:
I am not sure if this is our place but maybe we should warn that these are non-standard journals and the math is questionable?
Maybe something "These articles were published outside of the machine learning community, and might not have received the same amount of scrutiny as other sources"? Is that to harsh for :hankey:?